CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,7 @@
+Fix: modified engine version >=6 in packages.json
+Fix: update logops dependency from 1.0.8 to 2.1.0
+Add: config.http.timeout (and associated enviroment variable IOTA_HTTP_TIMEOUT)(#152)
+Add: config.mqtt.{cert,key,protocol,rejectUnauthorized} (and associated environment variables IOTA_MQTT_*)(#372)
+Add: readding sinon as dev dependency (~6.1.0)
+Upgrade NodeJS version from 8.12.0 to 8.15.0 in Dockerfile due to security issues
+Set Nodejs 8 as minimum version in packages.json (effectively removing Nodev6 from supported versions)
```

Comment:
I'm afraid the conflict in CHANGES_NEXT_RELEASE has not been correctly solved. Note that at the present moment that file is empty in master (see https://github.com/telefonicaid/iotagent-json/blob/master/CHANGES_NEXT_RELEASE). Thus, the changes in this file in this PR are only the new things added by this PR.

In sum, this file should contain only one line:

```
Set Nodejs 8 as minimum version in packages.json (effectively removing Nodev6 from supported versions) 
```

Reply:
Fixed by https://github.com/telefonicaid/iotagent-json/pull/400/commits/def895a461a0e867c8f965dc1f96da884c23efcc